resources/views/home/landing.blade.php

```diff
@@ -69,68 +103,91 @@ class="landing-nav__link js-nav-toggle"
+                    @if (present(config('osu.landing.video_id')))
+                        <iframe
+                            class="landing-hero__bg"
+                            src="https://www.youtube.com/embed/{{ config('osu.landing.video_id') }}?autoplay=1&controls=0&disablekb=1&loop=1&playlist={{ config('osu.landing.video_id') }}&modestbranding=1&enablejsapi=1&origin={{ config('app.url') }}"
+                            frameborder="0"
+                        ></iframe>
+                    @endif
+                </div>
+            </div>
 
-                        <span class="landing-hero__cta">
+            <div class="landing-hero__pippi">
+                <div class="landing-hero__pippi-logo"></div>
+            </div>
 
-                            <span class="landing-hero__content">
-                                {!! trans("home.landing.slogans.$i") !!}
-                            </span>
-                        </span>
-                    </span>
-                @endfor
+            <div class="landing-hero__info">
+                {!! trans("home.landing.players", ['count' => number_format($stats->totalUsers)]) !!},
+                {!! trans("home.landing.online", [
+                    'players' => number_format($stats->currentOnline),
+                    'games' => number_format($stats->currentGames)]
+                ) !!}
             </div>
 
-            <div class="landing-download">
-                <div class="landing-download__inner">
-                    <a href="http://m1.ppy.sh/r/osu!install.exe" class="landing-download__button shadow-hover">
-                        <span class="fa fa-cloud-download landing-download__icon"></span>
-                        <span class="landing-download__content-wrapper">
-                            <span class="landing-download__content landing-download__content--top">
-                                {{ trans("home.landing.download._") }}
+            <div class="landing-hero__messages">
+                <div class="landing-hero__slogan">
+                    <h1 class="landing-hero__slogan-main">
+                        {{ trans('home.landing.slogan.main') }}
+                    </h1>
+
+                    <h2 class="landing-hero__slogan-sub">
+                        {{ trans('home.landing.slogan.sub') }}
+                    </h2>
+                </div>
+
+                <div class="landing-hero__download">
+                    <a href="{{ config('osu.urls.installer') }}" class="btn-osu-big btn-osu-big--download">
+                        <span class="btn-osu-big__content">
+                            <span class="btn-osu-big__left">
+                                <span class="btn-osu-big__text-top">
+                                    {{ trans("home.landing.download._") }}
+                                </span>
+
+                                <span class="btn-osu-big__text-bottom js-download-platform"></span>
```

Comment:
Probably worth just having the button say "Download now for Windows" for now, instead of "...for [os]" and then linking to a Windows installer.